Incorrect winapi features?

In order to get this project compiling on windows I had to explicitly set winapi as my dependency and add "winerror" to the list of features. Otherwise you get the error:

```
error[E0432]: unresolved import `winapi::shared::winerror`
  --> C:\Users\travis\.cargo\registry\src\github.com-1ecc6299db9ec823\subprocess-0.1.16\src\win32.rs:22:25
   |
22 | pub use winapi::shared::winerror::{ERROR_BAD_PATHNAME, ERROR_ACCESS_DENIED};
   |                         ^^^^^^^^ Could not find `winerror` in `shared`
```

The correct config for me was:
```
[target.'cfg(windows)'.dependencies]
winapi = { version = "0.3.6", features = ["handleapi", "namedpipeapi", "processenv", "synchapi", "winerror"] }
```

Is this expected?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ libc = "0.2.43"
 
 [target.'cfg(windows)'.dependencies]
 crossbeam-utils = "0.5"
-winapi = { version = "0.3.6", features = ["handleapi", "namedpipeapi", "processenv", "synchapi"] }
+winapi = { version = "0.3.6", features = ["handleapi", "namedpipeapi", "processenv", "synchapi", "winerror"] }
 
 [dev-dependencies]
 tempdir = "0.3.7"

